lib/src/main/java/com/orange/ods/compose/component/progressindicator/OdsCircularProgressIndicator.kt

```diff
@@ -35,20 +41,47 @@ import com.orange.ods.utilities.extension.orElse
 @OdsComponentApi
 fun OdsCircularProgressIndicator(
     modifier: Modifier = Modifier,
+    label: String? = null,
```

Comment:
The Kdoc is missing for this new parameter. You can add an `@param label The label displayed below the circular progress.`